src/components/Reports/FinancialAccountsReport/AccountSummary/AccountSummary.tsx

```diff
@@ -0,0 +1,384 @@
+                    return (
+                      <StyledTableCell key={`${idx}-${balance}`} align="right">
+                        {numberFormat(balance, locale)}
+                      </StyledTableCell>
+                    );
+                  })}
+                </TableRow>
+                {/* Income header */}
+                <TableRow sx={{ background: theme.palette.mpdxGreen.main }}>
+                  <StyledTableCell>
+                    <Typography variant="body1">{t('Income')}</Typography>
+                  </StyledTableCell>
+                  {tableData.periods?.map((_, idx) => {
+                    return (
+                      <StyledTableCell
+                        key={`income-${idx}`}
+                        align="right"
+                      ></StyledTableCell>
+                    );
+                  })}
+                </TableRow>
+
+                {/* Income Categories */}
+                {tableData.creditsCategories?.map((category, idx) => {
+                  return (
+                    <TableRow key={`${idx}-${category.name}`}>
+                      <StyledTableCell component="th" scope="row">
+                        <Typography variant="body1" width="250px">
+                          {category.name}
+                        </Typography>
+                      </StyledTableCell>
+
+                      {category.amounts.map((amount, idx) => (
+                        <StyledTableCell key={`${idx}-${amount}`} align="right">
+                          {numberFormat(amount, locale)}
+                        </StyledTableCell>
+                      ))}
+                    </TableRow>
+                  );
+                })}
+
+                {/* Total Income */}
+                <TableRow>
+                  <StyledTableCell>
+                    <Typography variant="body1" fontWeight="bold">
+                      {t('Total Income')}
+                    </Typography>
+                  </StyledTableCell>
+                  {tableData.credits?.map((credit, idx) => {
+                    return (
+                      <StyledTableCell key={`${idx}-${credit}`} align="right">
+                        <Typography fontWeight="bold">
+                          {numberFormat(credit, locale)}
+                        </Typography>
+                      </StyledTableCell>
+                    );
+                  })}
+                </TableRow>
+
+                {/* Expenses header */}
+                <TableRow sx={{ background: '#FF7679' }}>
+                  <StyledTableCell sx={{ color: theme.palette.common.white }}>
+                    <Typography variant="body1">{t('Expenses')}</Typography>
+                  </StyledTableCell>
+                  {tableData.periods?.map((_, idx) => {
+                    return (
+                      <StyledTableCell
+                        key={`expenses-${idx}`}
+                        align="right"
```

Comment:
Also empty

Reply:
This table row needs styles and empty rows to go all away across the table

Reply:
> This table row needs styles and empty rows to go all away across the table

Definitely! I'm just wondering why we have `align="right"` on an empty cell.